content/desktop/synchronized-file-sharing.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Synchronized file sharing
+description: Get started with Synchronized file sharing on Docker Desktop.
+sitemap: false
+---
+
+> **Early Access**
+>
+> Synchronized file sharing is an early-access feature. 
+>
+> If you would like to be considered for Synchronized file sharing testing, you can
+> [sign up for the early access program](https://www.docker.com/build-early-access-program/?utm_source=docs).
+{ .restricted }
+
+Synchronized file sharing is an alternative file sharing mechanism powered by [Mutagen](https://mutagen.io/). It provides fast and flexible host-to-VM file sharing by replacing bind mounts with synchronized filesystem caches. 
+
+![Image of Synchronized file shares pane](images/synched-file-sharing.png)
+ 
+## Who is it for?
+
+For developers who: 
+- Have large repositories or monorepos with 100 000 files or more totaling hundreds of megabytes or even gigabytes
+- Are using virtual filesystems, such as VirtioFS, gRPC FUSE, and osxfs, which are no longer scaling well with their codebases 
+- Regularly encounter performance limitations
+- Don't want to worry about file ownership or spend time resolving conflicting file-ownership information when modifying multiple containers
+
+## How does Synchronized file sharing work?
+
+A Synchronized file share behaves just like a virtual file share, but takes advantage of Mutagen's high-performance, low-latency code synchronization engine to create a synchronized cache of the host files on an ext4 filesystem within the Docker Desktop VM. If you make filesystem changes on the host or in the VM’s containers it propagates via bidirectional synchronization.
+
+Synchronized file sharing also adds an additional Docker Context, `desktop-linux-mutagen`, which uses Docker socket middleware to automatically replace bind mounts with in-VM caches whenever possible.
+
+After creating a file share instance, any container using a bind mount that references a host filesystem location which matches the stated synchronized file share host location, is provided by the Synchronized file sharing feature. Bind mounts that don't satisfy this condition are passed to the normal virtual filesystem bind-mounting mechanism, for example VirtioFS or gRPC-FUSE.
+
+> **Important**
+>
+> Synchronized file sharing isn't available on WSL or when using Windows containers. 
+{ .important }
+
+## Set up
+
+Before you get started with Synchronized file sharing, you need to:
+
+- Download and install [Docker Desktop version 4.25](release-notes.md)
+- Have a Pro, Team, or Business subscription
+- Have a Docker ID that's part of a Docker organization participating in the Early access program
```

Comment:
Technically being part of a Docker organization in the EAP isn't a requirement; we're now just using a hidden activation mechanism (with the Pro/Team/Business requirement) rather than an entitlement mechanism.

@smnovick should we remove this?  It might keep users "in-line" and avoid them sharing the instructions (but TBH I think we're okay if this gets shared around a bit).